synco.py

```diff
@@ -48,26 +48,29 @@ def stop(self):
         self.done = True
         self.join()
 
-
-class MidiOut(): # check docs for this
+class MidiOut(): 
     '''
     Will instatiate the drumseq.Sequencer with the
     parsed pattern from user input (shell).
     and play notes when .play() is called
     '''
 
-    def __init__(self, input_pattern=None, port_desc=None, **kwargs):
+    def __init__(self, input_pattern=None, drumseq_pattern=None, port_desc=None, **kwargs):
         '''
         This will take most of the functionality of __enter__
         i.e. - it will load an instance of the seqquencer with the pattern.
         Play will actually call sequencer (see below)
         '''
 
-        if not input_pattern:
+        if not (input_pattern or drumseq_pattern):
```

Comment:
To me, this if indicates you could specify one OR the other and not encounter an error.  Is this what you intended?